web/sites/default/settings.php

```diff
@@ -804,51 +804,54 @@
  * Cloud Foundry places all service credentials in VCAP_SERVICES
  */
 $cf_application_data = json_decode($_ENV['VCAP_APPLICATION'] ?? '{}', TRUE);
-$SERVER_HTTP_HOST = $_SERVER['HTTP_HOST'];
-if (!empty($cf_application_data['space_name']) &&
-    in_array($cf_application_data['space_name'],
-             ['dev', 'stage', 'prod'])) {
-  switch (strtolower($cf_application_data['space_name'])) {
-    case "dev":
-      $SERVER_HTTP_HOST = 'https://cms-dev.usa.gov';
-      break;
-
-    case "stage":
-      $SERVER_HTTP_HOST = 'https://cms-stage.usa.gov';
-      break;
+$IS_CLOUDGOV = FALSE;
+$SERVER_HTTP_HOST = $_SERVER['HTTP_HOST'] ?? 'cms-dev.usa.gov';
+$settings['trusted_host_patterns'] = [];
+$space_name = strtolower($cf_application_data['space_name'] ?? '');
 
-    case "prod":
-      $SERVER_HTTP_HOST = 'https://cms.usa.gov';
+if (in_array($space_name, ['local', 'dev', 'dr', 'stage', 'prod'], true)) {
+  switch (strtolower($cf_application_data['space_name'])) {
+    case "local":
+      $settings['trusted_host_patterns'][] = '^cms-local.usa.gov$';
+      $settings['trusted_host_patterns'][] = '^cms-local-usagov.apps.internal$';
+//      $config['core.extension']['module']['devel'] = 0;
+//      $config['core.extension']['module']['devel_generate'] = 0;
       break;
-  }
-}
 
-$IS_CLOUDGOV=FALSE;
-$SERVER_HTTP_POST = $_SERVER['HTTP_HOST'] ?? 'cms-dev.usa.gov';
-$cf_application_data = json_decode($_ENV['VCAP_APPLICATION'] ?? '{}', TRUE);
-if (!empty($cf_application_data['space_name']) &&
-    in_array($cf_application_data['space_name'],
-             ['dev', 'stage', 'prod'])) {
-  switch (strtolower($cf_application_data['space_name'])) {
     case "dev":
-      $IS_CLOUDGOV=TRUE;
+    case "dr":
```

Comment:
"dr" needs to have trusted_host_patterns '^cms-dr.usa.gov$' and '^cms-dr-usagov.apps.internal$'

(I was scratching my head over how cms-dr.usa.gov was working at all, but I've figured out that we simply don't have trusted host settings enabled there currently.)